modules/vaos/app/schemas/appointments_index.json

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
This schema was created based on [this documentation](https://veteran.apps.va.gov/vaos/v1/).